### PR TITLE
Model is a fine name, but it is two syllables

### PR DESCRIPTION
**Pull request by jaresty**
- some people may want to rename their model to avoid conflicts with their particular configuration it or to make it easier for them to say the name

### CODE REVIEW

File: GPT/lists/modelName.talon-list [hunk 1]
```diff
@@ -0,0 +1,5 @@
+list: user.modelName
+-
+
+# The default model name
+model: model
```
Comment (jaresty) [inline GPT/lists/modelName.talon-list:5]: aegis recommended using a talon list for this even though it is a single word.

**Comment by C-Loftus** [comment]
This is OK but it hurts readability for the commands. Now that we have the modifier as well I think it is confusing for new users. 

New users can't look at the talon files anymore to get a sense of what the commands do. 

Model name as a list is a bit confusing since it implies we could specify a different model on the fly which is not the case. 

I think we need to improve documentation more before we merge this.

**Comment by jaresty** [comment]
You can't override this locally unless you modify the repository-it also requires modifying in several places. I really wanted to experiment with some different model names-I find the word model really difficult to say. Can we improve the documentation to make this possible to merge?

**Comment by jaresty** [comment]
Fwiw, I think there is a possible future feature to support multiple models in the fly as well since there are various reasons to prefer one or the other and it's currently difficult to switch here to experiment. The continue extension, for example, does support switching in the fly. I don't want to expand the scope of this PR but I wanted to point out that we may need some grammar to switch eventually.

**Comment by jaresty** [comment]
To give you a sense of some of the names I'm considering:
- skip
- brain
- jeep
- pete
- bird
- dog
- ...


I think it's obvious why some of these aren't sensible to upstream. I'm using "model" as often or more often than I'm using cursorless commands and even letter names, so having some way to customize the name is really becoming an issue for me. Can you think of another way to customize the name that does not require forking?

**Comment by C-Loftus** [comment]
I am ok with merging this and compromising if we can get rid of the model modifiers list as you mentioned in slack. Not sure if you have seen the docs site and how we are generating it. My goal is to have a docs as code setup. At build time the docs directly fetch the source files and output them in the docs. Hence why I want to have very clear and annotated `.talon` files. This makes it so our docs are hopefully always in sync with the state of the repo. The more modifiers / lists we have in the commands, the harder it is for a new user to understand how to use the command. Does that seem reasonable where I am coming from? I still am trying to focus on improving UX for new users.

**Comment by jaresty** [comment]
Totally! I'm going to get started on replacing the modifiers with destinations next. I think that will be a good improvement.

**Comment by jaresty** [comment]
I think I wrapped up removing the modifiers - let me know if you see any issues. I updated this pull request based on the new changes, so hopefully this is now ready to review.

**Comment by C-Loftus** [comment]
Looks good to me with two minor comments.

1. I wonder if should just use `user.model` as the name for the list. Or perhaps `user.modelName` or `user.modelKeyword`. Honestly not sure if these are more clear. Trying to think which way of structuring it is most clear for a new user who will be reading the `.talon` file and needs to understand that the keyword `model` is a part of that list. 
2. I added this documentation to the docs site and removed it from the readme. I think we can probably just use the readme for very essential info and otherwise put everything else in the docs website.

**Comment by jaresty** [comment]
I like the idea of "user.model" as the name- then the default reads the same as before.

**Comment by C-Loftus** [comment]
Looks good to me. And in the future if needed we can switch to `modelName` if users fine this confusing or it conflicts with another list for some reason.

**Pull request merged**